examples/experimental/export_example.ipynb

```diff
@@ -0,0 +1,146 @@
+{
```

Comment:
<p>what does this output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/truera/trulens/pull/1346/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>The spans that were recorded during the invocations above.</p>

Reply:
Would be good to keep output for this cell to show what the spans look like

Reply:
Need to figure out how to keep outputs in notebooks as the recent packaging change clears all notebook outputs.

Reply:
For docs at least.

Reply:
Ohh, looks like the clearing is done in the pre-commit hooks.